CHANGELOG.md

```diff
@@ -1,6 +1,27 @@
 Note: This is in reverse chronological order, so newer entries are added to the top.
 
 
+Swift 5.3.1
+-----------
+
+* [#2937](https://github.com/apple/swift-package-manager/pull/2937)
+  
+  Manifest files can now have any combination of leading line terminators. All [Unicode line terminators](https://www.unicode.org/reports/tr14/) are recognised. Gone are ye olde days when a shebang-like directive must be at the start of file.
+  
+  [SR-13566](https://bugs.swift.org/browse/SR-13566) The Swift tools version specification in each manifest file now accepts any combination of _horizontal_ whitespace characters between `//` and `swift-tools-version`, if and only if the specified version > 5.3. For example, `//swift-tools-version:5.3.1` and `//		 swift-tools-version:5.4` are valid.
+  
+  * Deprecations
```

Comment:
One of the things I marked as deprecated is [`ToolsVersionLoader.Error.malformedToolsVersion(_:_:)`](https://github.com/WowbaggersLiquidLunch/swift-package-manager/blob/1ef5badfd623827c8176acea0cc760ddfb8eaa7e/Sources/PackageLoading/ToolsVersionLoader.swift#L130). Since `libSwiftPM` isn't API-stable, should I remove the deprecated case instead?

Reply:
I think that makes sense.  At some point we will need to review the API and then start supporting it from there (including deprecating instead of removing, etc), but at present we are still in prerelease mode for libSwiftPM.

Reply:
In commit 2e753b88fbc59ca9fdc6a798017b667c72bb8fad, I removed `ToolsVersionLoader.Error.malformedToolsVersion(_:_:)` instead of marking it as deprecated. 

In commit d433487e6461821c4e7d2c8cbbac4cdc2f0c1252, I moved the notice of `ToolsVersionLoader.Error.malformedToolsVersion(_:_:)` from deprecation to removal, and removed the notice of `ToolsVersionLoader.split(_ bytes: ByteString) -> (versionSpecifier: String?, rest: [UInt8])`'s deprecation.